src/include/duckdb.h

```diff
@@ -878,6 +878,19 @@ typedef struct _duckdb_catalog_entry {
 	void *internal_ptr;
 } * duckdb_catalog_entry;
 
+//===--------------------------------------------------------------------===//
+// Logging Types
+//===--------------------------------------------------------------------===//
+
+//! Holds a log storage object.
+typedef struct _duckdb_log_storage {
+	void *internal_ptr;
+} * duckdb_log_storage;
+
+//! This function is missing the logging context which may be added later, or in a secondary function.
```

Comment:
Maybe we can emphasize in a comment that the logger C API is, in its current state, not suitable for being made stabilized?